go.mod

```
@@ -0,0 +1,17 @@
+module weaveTest
```

generally module and package names don't use camel case
```suggestion
module weave-test
```